packages/web-api-client/src/a11y-service-client.ts

```diff
@@ -43,9 +43,29 @@ export class A11yServiceClient {
     }
 
     public async postScanUrl(scanUrl: string, priority?: number): Promise<ResponseWithBodyType<ScanRunResponse[]>> {
-        const requestBody: ScanRunRequest[] = [{ url: scanUrl, priority: priority === undefined ? 0 : priority }];
+        const requestBody: ScanRunRequest = { url: scanUrl };
+
+        return this.postScanUrlWithRequest(requestBody, priority);
+    }
+
+    public async postConsolidatedScan(
+        scanUrl: string,
+        reportId: string,
+        priority?: number,
+    ): Promise<ResponseWithBodyType<ScanRunResponse[]>> {
+        const requestBody: ScanRunRequest = {
```

Comment:
Same comment here

Reply:
changes show in most second most recent commit